src/signatures/commitment_signature.rs

```diff
@@ -159,14 +183,31 @@ where
         &self.public_nonce
     }
 
-    /// Returns a canonical byte representation of the commitment signature
+    /// Returns a canonical vector of bytes representation of the commitment signature
```

Comment:
A vector of bytes would be `Vec<bytes>` where `bytes` is probably `Vec<u8>` :)

Reply:
Nice catch eheh :) did refactor accordingly